casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -58,11 +61,19 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
 
   it should "handle multi-parent blocks correctly when they operate on stdout" ignore effectTest {
     def echoContract(no: Int) = s"""new stdout(`rho:io:stdout`) in { stdout!("Contract $no") }"""
-    val time                  = System.currentTimeMillis()
+    testConflictOnSimpleDiamondDag(echoContract)
+  }
+
+  it should "handle multi-parent blocks correctly when they operate on volatile produce/consume pairs" in effectTest {
+    testConflictOnSimpleDiamondDag((no: Int) => s"""@"hi"!($no) | for (_ <- @"hi") { Nil }""")
```

Comment:
I might misunderstand the concept of "volatile" but, is this volatile, since the messages are sent to a forgeable channel?

Reply:
Volatile just mean that it doesn't affect the trie at the end. Volatility depends on the execution - if someone else sent to the forgeable name then it may not be volatile.